### ISSUE TITLE
27c160

### ISSUE THREAD

**Issue by Indibilio** [soniccd123/Genesis-FeRAM-Cart#5]
Hello! How i can use a 27c160 EEPROM whit this cartridge? Which modifications it needs? 

Thanks.

**Comment by soniccd123** [comment]
Hello! Yes it should work, just put turn A20 off in the switch (switch photo with A20 marked for reference is attached to this message), this will garantee that the 27C160 Byte pin is high, setting it to 16bit mode and will disconnect this pin from the Genesis Cart bus.

<img width="378" alt="switch" src="https://user-images.githubusercontent.com/54874140/115944081-1635d180-a48a-11eb-9c41-4ecc5dc0238a.png">

**Comment by Indibilio** [comment]
Thanks for the reply. I thought I had to cut a track. So I turn OFF that switch in that position, and the cartridge already works with EEPROM 27c160 and lower.

Great job!!!

**Comment by Indibilio** [comment]
Hello. Another question. The FM1808 is discontinued and its price is increased.

It is possible to use another equivalent fram? Or if i don't use the 3 top switch to use another memory banks, it is possible to use a small fram memory type? Only one game - one save bank.

Thanks!!!!☺️☺️

**Comment by soniccd123** [comment]
Yes, you can use a equivalent as long as it is pin compatible and 5V tolerant. I personally use the FM18W08 which is mostly identical and is widely available.

You can also use a smaller FeRAM if you want, but need to check if it is pin compatible, I don't really have any chips to recommend, but they surely exist.

**Comment by Indibilio** [comment]
Thanks.

It is dificult to modiffy this pcb to allow two 27c322 and put the rom of "Pier Solar" inside? Using the mapper to play a game with 64mb with save feature. This game is sold out, not disponible, and dificult to play into a real hardware.

**Comment by soniccd123** [comment]
Hey! Sorry for the making you wait so much, life got in the way and its been some time i didn't check my github.

Well, it is perfectly possible if you can implement the mapper. I really don't know much about how Pier Solar maps the 64Mb, is it a SSF mapper? Really depends on how much space the circutry takes, but should be doable.

**Comment by Indibilio** [comment]
I think it will be complicated, I don't know how the Pier Solar mapper works, I only have a slight knowledge of electronics. I thought that using two 27c322s together with a traditional mapper could be done. The Chinese on Aliexpress do it, but I don't know how.

Courage, life sometimes gets difficult, but don't forget to live !!!

**Comment by PicoElements** [comment]
Hi! I love this project! I'm trying to make it work with a 27c160 and i saw the first comment so very helpful! All the switches on but the A20. Cn it work without the FM18W08 ? Is there a way not to use it? It's really hard to find.
Thanks a lot!

**Comment by Indibilio** [comment]
You can find the FM18W08 or FM1808 from Aliexpress....

**Comment by melvimchang** [comment]
Hi, I can tell you that It os a great project. I already did 3 types off cartridges with thia PCB. Multicart, 27c322 FRAM and 27c160 FRAM. The Cartridges with 27c160 for se reason is not saving. I already tested the two position of JP1. For example, Shining force 2. JP1 ser to Rom <2MB the starring game os unstable. It ramdonly gets stuck when you select Save 1 or 2. If I set JP1 to >2MB, the games shows "garbage".  Is there something that can be done to fix It? I also having problem with Monster World IV and Landstalker
